model/src/main/java/com/epam/esm/model/parameters/CertificateSearchParameter.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.esm.model.parameters;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum CertificateSearchParameter {
```

Comment:
this class is not used - why it's in the PR?

Reply:
done